api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractServiceImpl.java

```diff
@@ -19,10 +19,10 @@
     @Override
     public ContractModel createContract(@NonNull final UserModel proposer, @NonNull final UserModel streamer, @Nullable final String game, @NonNull final String description) {
         final Timestamp creationTimestamp = new Timestamp(System.currentTimeMillis());
-        // Add 10 days to the current timestamp to create the expires timestamp
+        // Add 3 days to the current timestamp to create the expires timestamp
         final Calendar calendar = Calendar.getInstance();
         calendar.setTime(creationTimestamp);
-        calendar.add(Calendar.DAY_OF_WEEK, 10);
+        calendar.add(Calendar.DAY_OF_WEEK, 3);
```

Comment:
Make this a constant? I generally think its bad practice to have numbers hard coded in the code.

Reply:
Sure.... I wonder who added the magic number in the first place ;)

Reply:
Probably me :)